google_analytics/ga_report.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "HSMR public dashboard - Google analytics report"
+author: "Public Health Scotland"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output: 
+  html_document:
+    toc: true
+    toc_float: true
+---
+  
+  ```{r TODO, include = FALSE}
```

Comment:
It moans about the indentation here so might be worth updating so the message doesn't appear again?